jobs/jobs.go

```diff
@@ -32,6 +33,7 @@ type Job struct {
 	Type                   string         `gorm:"column:type"`
 	State                  State          `gorm:"column:state;default:INIT"`
 	Error                  string         `gorm:"column:error"`
+	Errors                 pq.StringArray `gorm:"column:errors;type:text[]"`
```

Comment:
Left the `Error` field intact, as that's a quick way to check if the final execution ended in an error. Thoughts on `Errors` vs. `ErrorHistory`, other suggestions?